### ISSUE TITLE
Compose Ambient mode support

### ISSUE THREAD

**Issue by yschimke** [google/horologist#198]
Consider what a Compose Ambient mode support library would include, looking at https://github.com/android/wear-os-samples/pull/189

**Comment by yschimke** [comment]
Blocked on documenting the desired behaviour for apps.

**Comment by yschimke** [comment]
Out of scope for now.
